modules/ROOT/pages/dw-arrays.adoc

```diff
@@ -10,13 +7,34 @@ by adding the line `import * from dw::core::Arrays` to the header of your
 DataWeave script.
 
 == Functions
+
+[%header, cols="1,3"]
 |===
-| Name
-| xref:dw-arrays-functions-countby.adoc[countBy]
-| xref:dw-arrays-functions-divideby.adoc[divideBy]
-| xref:dw-arrays-functions-every.adoc[every]
-| xref:dw-arrays-functions-some.adoc[some]
-| xref:dw-arrays-functions-sumby.adoc[sumBy]
+| Name  | Description
+| xref:dw-arrays-functions-countby.adoc[countBy] | Counts the elements in a list (array) that match the results of a function.
+| xref:dw-arrays-functions-divideby.adoc[divideBy] | Breaks up a list (array) into sub-lists (sub-arrays) that contain the
+specified number of elements.
+| xref:dw-arrays-functions-drop.adoc[drop] | Drops first `n` elements. It returns the original array when `n &lt;= 0` and an empty array when `n > sizeOf(array)`.
+| xref:dw-arrays-functions-dropwhile.adoc[dropWhile] | Drops elements from the array while the condition is met.
+| xref:dw-arrays-functions-every.adoc[every] | Returns `true` if every element in the array matches the condition.
+| xref:dw-arrays-functions-indexof.adoc[indexOf] | Returns the index of the first occurrence of an element within the array.
+| xref:dw-arrays-functions-indexwhere.adoc[indexWhere] | Returns the index of the first occurrence of an element that matches a condition within the array.
+| xref:dw-arrays-functions-join.adoc[join] | Joins two array of objects by a given `ID` criteria.
+It will return an Array of object containing only the ones that the `ID` of the `left` can be matched with an `ID` of the `right`.
```

Comment:
I suggest: It returns an array of objects containing only those where the ID of the left matches the ID of the right.